src/server/src/main/java/io/cassandrareaper/service/RepairRunService.java

```diff
@@ -189,9 +189,9 @@ public RepairRun registerRepairRun(
   static int computeGlobalSegmentCount(
       int segmentCountPerNode,
       Map<String, List<RingRange>> endpointToRange) {
-    Preconditions.checkArgument(1 <= endpointToRange.keySet().size());
+    //Preconditions.checkArgument(1 <= endpointToRange.keySet().size());
```

Comment:
why is this now ok?

Reply:
It was just a leftover from an attempt to run integration tests that didn't go through.
I've reverted this change.